pagopa/io-app
# chore: [#178393622] Changed #5C6F82 color to #475A6D

## Short description
Changed bluegrey color from #5C6F82 to #475A6D to pass accessibility test

## How to test
Move to a place in the app where the #5C6F82 color was used and check if the new color is used correctly

## Example:

**Before:**
<img width="300" alt="Screenshot 2021-06-04 at 18 47 28" src="https://user-images.githubusercontent.com/61834253/120836012-5f873f80-c565-11eb-8c17-7c569d854cfd.png">

**After:**
<img width="300" alt="Screenshot 2021-06-04 at 18 44 19" src="https://user-images.githubusercontent.com/61834253/120835848-25b63900-c565-11eb-8ca2-8cc9ced6742e.png">
